static/skin/index.js

```diff
@@ -114,12 +114,18 @@
                 ${downloadLink ? `<div class="book__download"><span data-link="${downloadLink}">Download ${humanFriendlyZimSize ? ` - ${humanFriendlyZimSize}</span></div>`: ''}` : ''}
             </div>
             <div class="book__description" title="${description}">${description}</div>
-            <div class="book__languageTag" ${languageAttr}>${language.substr(0, 2).toUpperCase()}</div>
+            <div class="book__languageTag" ${languageAttr}>${getLanguageCode(language)}</div>
```

Comment:
I see nowhere `iso6391To3.js` included properly in the HTML. Are you sure this works?

Reply:
Included in `index.html` line 42
Used previously in `index.js` line 412